modules/ROOT/pages/design-named-examples.adoc

```diff
@@ -208,9 +208,28 @@ types:
     properties:
       givenName: string
       familyName: string
-   example: !include fragment1.raml
+   example: !include fragment1.yaml
 ----
 
+.fragment1.yaml
+----
+givenName: ”Chiaki”
+familyName: "Mukai"
+----
+
+.fragment2.yaml
+----
+givenName: "Kyung-won"
+familyName: "Park"
+----
+
+
+[NOTE]
+====
+According to the RAML 1.0 specification, a NamedExample fragment that defines only one example is not valid. Therefore, you should not use NamedExample fragments when you are using single, unnamed examples.
+
+For example, the following NamedExample fragments would be invalid:
```

Comment:
would be invalid > are invalid

GLOBALLY standardize to simple present tense as much as possible without impairing meaning; if present works just as well, try to use it. It's so much easier to translate.